docs/csharp/language-reference/operators/patterns.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: "Patterns - C# reference"
-description: "Learn about the patterns supported by C# pattern matching expressions and statements - C# reference"
+description: "Learn about the patterns supported by C# pattern matching expressions and statements"
```

Comment:
What are the guidelines about description? Should they end in a period or not?
Currently the PR uses both.

Reply:
I don't know. I've noticed some have a period, others - not. I've decided to ignore that for now. However, if anyone knows what should be, I'll update the articles in this folder. @gewarren do you know by chance?

Reply:
I looked this morning. The contributor guide doesn't say one way or the other. I think we've been adding the period, as they are typically complete sentences. (That's the standard we follow in bullet lists: The items should either all be sentences, or all not be sentences, and the punctuation should match).

Reply:
Thank you @BillWagner I've applied your suggestions and also checked other operator articles to ensure descriptions end with a period.

Reply:
I don't think it really matters for search engine purposes, but I'd say it looks better with a period since it's a complete sentence. From the SEO cheat sheet:

![image](https://user-images.githubusercontent.com/24882762/116312843-da3c8e00-a761-11eb-8f30-c7d2d2d42491.png)